.github/workflows/build-wheels.yml

```diff
@@ -96,14 +96,14 @@ jobs:
         with:
           fetch-depth: 0
       - name: Set up Python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v4
         with:
-          python-version: 3.8
+          python-version: "3.8"
```

Comment:
Here you use python 3.8 but in other files, such as .github/workflows/tests.yml, it says python 3.10. Should python versioning be consistent everywhere?

Reply:
Yes we can try it. At least the manylinux we use here should work for 3.10